src/cozy/services/installation-guard.service.ts

```diff
@@ -1,89 +1,89 @@
-import { Injectable } from '@angular/core';
-import {
-    ActivatedRouteSnapshot,
-    CanActivate,
-    RouterStateSnapshot,
-} from '@angular/router';
-import { MessagingService } from '../../../jslib/src/abstractions/messaging.service';
-import { CozyClientService } from './cozy-client.service';
-
-// @ts-ignore
-import flag from 'cozy-flags';
-import { FORCE_VAULT_UNCONFIGURED } from '../flags';
-
-@Injectable({ providedIn: 'root' })
-export class VaultInstallationService {
-    userFinishedInstallation = false;
-
-    constructor(private clientService: CozyClientService) {}
-
-    async IsVaultInstalled(): Promise<boolean> {
-        try {
-            const client = this.clientService.GetClient();
-            const vault = await client.stackClient.fetchJSON(
-                'GET',
-                '/data/io.cozy.settings/io.cozy.settings.bitwarden',
-                []
-            );
-
-            return !flag(FORCE_VAULT_UNCONFIGURED) && (vault.extension_installed || this.userFinishedInstallation);
-        } catch {
-            return false;
-        }
-    }
-
-    async completeVaultConfiguration() {
-        const client = this.clientService.GetClient();
-        await client.stackClient.fetchJSON(
-            'POST',
-            '/settings/vault'
-        );
-        this.userFinishedInstallation = true;
-    }
-}
-
-@Injectable({ providedIn: 'root' })
-export class VaultInstalledGuardService implements CanActivate {
-    constructor(
-        private vaultInstallationService: VaultInstallationService,
-        private messagingService: MessagingService
-    ) {}
-
-    async canActivate(
-        route: ActivatedRouteSnapshot,
-        routerState: RouterStateSnapshot
-    ) {
-        const isInstalled =
-            await this.vaultInstallationService.IsVaultInstalled();
-
-        if (!isInstalled) {
-            this.messagingService.send('installBlocked');
-            return false;
-        }
-
-        return true;
-    }
-}
-
-@Injectable({ providedIn: 'root' })
-export class VaultUninstalledGuardService implements CanActivate {
-    constructor(
-        private vaultInstallationService: VaultInstallationService,
-        private messagingService: MessagingService
-    ) {}
-
-    async canActivate(
-        route: ActivatedRouteSnapshot,
-        routerState: RouterStateSnapshot
-    ) {
-        const isInstalled =
-            await this.vaultInstallationService.IsVaultInstalled();
-
-        if (isInstalled) {
-            this.messagingService.send('uninstallBlocked');
-            return false;
-        }
-
-        return true;
-    }
-}
+import { Injectable } from '@angular/core';
```

Comment:
Seems like you changed this file's EOL, is that expected? (nothing else changed)

Reply:
Fixed 👍